docs/cluster-autodeploy.md

```diff
@@ -0,0 +1,22 @@
+# Autodeploy
+
+This is a lightweight, automated way of deploying a virtualized cluster for testing.
+
+## Configuration
+
+To get an autodeploy configuration:
+
+    $ cp deploy-chroot/setup.yaml.in $HOMEWORLD_DIR
+    $ deploy-chroot/generate-setup.py <x>
+
+Here `<x>` is a byte which must be unique per machine
+if you plan to do multiple concurrent autodeploys.
+If you are working on rhombi, please coordinate assignment of these numbers
+via the `/lock` bot on mattermost.
+
+## Autodeploy
+
+    $ eval `ssh-agent -s` && ssh-add  # if not already running
```

Comment:
Might want to clarify what "if not already running" means for people who don't realize this will start a SSH agent?

Reply:
There is a more verbose explanation in the other doc, and I don't feel like explaining basic things every time in these documents.